### DISCUSSION TITLE
Using logging from r2r?

### DISCUSSION THREAD

**Discussion by otiv-milan** [ros2-rust/ros2_rust#405] [Ideas]
Would it be OK to "just" copy+adapt logging code from https://github.com/sequenceplanner/r2r/blob/master/r2r/src/utils.rs to this repo?

Saw that there is https://github.com/ros2-rust/ros2_rust/issues/184 but nothing actually happened so far.

**Comment by otiv-milan** [comment]
Implemented in my other repo https://github.com/mvukov/rules_ros2/pull/371, but I can transplant that here. @esteve @maspe36 thoughts?

**Reply by esteve** [reply]
Thanks for the work, could you create a PR against ros2_rust instead? Then we'd have to discuss with the authors of r2r if they'd be ok with us merging their code (not legally needed, but it's good etiquette) and perhaps spin it off to a separate crate if that's easier to maintain for both projects.
